README.md

```diff
@@ -1,373 +1,68 @@
-# Getting Started 
```

Comment:
um, should we keep some of the old README??

Reply:
Yes